deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/artillery.yml

```diff
@@ -0,0 +1,37 @@
+config:
+  # Dashboard Base URL
+  target: https://127.0.0.1/
```

Comment:
The target is currently hardcoded. To enhance the tool's versatility, dashboard tests should be able to target and saturate remote hosts.

Reply:
The target is always changed, it is a mandatory change in the Python script. I have removed the line from the YAML to avoid confusion. Solved in https://github.com/wazuh/wazuh-qa/pull/5629/commits/ca020ab5fd076f3ff705605233b10f1ca0921806